### DISCUSSION TITLE
Finding Files Giving an Error Message In Fresh Install Of LazyVim

### DISCUSSION THREAD

**Discussion by iammihirsig** [LazyVim/LazyVim#4028] [General]
- I just installed LazyVim. However I try to search files using `<space> + f + f ` It gives an error message.
![1](https://github.com/user-attachments/assets/d8bd45e9-2694-4d1a-b19f-2f887464cd56)
- I am new to vim and stuffs and learning things, however I don't know how to tackle this. Any help will be appreciated.

**Comment by amooozk** [comment]
I was having the same issue, after the update. Maybe the developers are working on something new. There may be some conflicts in the code which are giving this issue.. 
So, I figured out that the commit in LazyVim/ with the description "fix(pick): get rid of "auto" picker and set better defaults for telescope find_files (#4024)" has created the problem.
So, just shift git to the commit just before that...

I mean:
git checkout this commit of the previous day.
![image](https://github.com/user-attachments/assets/4a349e40-886b-414c-8988-67ca6233123b)

The final command becomes this :
`cd ~/.local/share/nvim/lazy/LazyVim/ && git checkout 1f8469a53c9c878d52932818533ce51c27ded5b6`

Remember to do this before updating the next time: (to avoid any errors)
`cd ~/.local/share/nvim/lazy/LazyVim/ && git checkout main`

NOTE: Please take note that this is only a temporary solution; Lazy.nvim will display the pending update, which will cause the issue to recur.For that reason, wait to update it until something new (that resolves the issue) arrives. Thus, until a new LazyVim update is released, I will stick with this commit.

**Reply by iammihirsig** [reply]
- Thank you for such brief information. I am new and looking forward to learning and work with LazyVim.I decided to choose LazyVim because, I don't want to waste so much time just figuring out minor configs itself. I will be grateful if I have some resources to learn.

- Also, What will LazyVim users do if something breaks? I mean, how not to hinder one's workflow?

**Reply by noahgsolomon** [reply]
thanks so much. This works. A shame this happened right after I changed my entire config last night, so I thought it was me lol

**Reply by dpetka2001** [reply]
Do you have [fd](https://github.com/sharkdp/fd) installed? LazyVim uses that instead of `ripgrep` or `find` for `find_files` picker. Alternatively, you could maybe put in your personal configuration 
```lua
-- ~/.config/nvim/lua/plugins/telescope.lua
return {
  "nvim-telescope/telescope.nvim",
  opts = {
    pickers = {
      find_files = {
        find_command = { "rg", "--files", "--color", "never" }
        -- find_command = { "find", ".", "-type", "f" }  -- use this if you don't have neither `rg` nor `fd` to use default `find`
      },
    },
  },
}
```

**Reply by amooozk** [reply]
Yeah, I have fd-find and ripgrep installed...
Hmm. I would try this configuration as well in the future, if the problem persists. 
Thanks... 😄

**Reply by dpetka2001** [reply]
fd-find is different from `fd`.

**Reply by iammihirsig** [reply]
> fd-find is different from `fd`.

I tried searching to install fd. But most of the time gets command related to fd-find.

However I will try explicitly define telescope.lua in plugins. I wanna see that if that works together with latest commit on main branch. Till developers fixed it.

**Reply by dpetka2001** [reply]
In my first comment I linked the github repo to `fd`. But here it is again https://github.com/sharkdp/fd

**Reply by folke** [reply]
https://github.com/LazyVim/LazyVim/commit/d39cdb059624b1200f2ef762f30c6e3b74066bc5

**Reply by iammihirsig** [reply]
> [d39cdb0](https://github.com/LazyVim/LazyVim/commit/d39cdb059624b1200f2ef762f30c6e3b74066bc5)

😄 It is fixed. Refer here.

**Comment by Flimm** [comment]
I had a similar problem. It turns out that I had installed `fdclone`, which is a console-based lightweight file manager (FD represents "File and Directory". I didn't realise that I had this executable installed, because I had configured my shell to make `fd` an alias of `fdfind`. To fix it, I uninstall `fdclone` by running:

    sudo apt remove fdclone
